mockk/src/main/kotlin/io/mockk/impl/annotations/JvmMockInitializer.kt

```diff
@@ -55,6 +58,17 @@ class JvmMockInitializer(val gateway: MockKGateway) : MockKGateway.MockInitializ
                     arrayOf()
                 )
             }
+
+            if (property.findAnnotation<InjectMockKs>() != null) {
+                val injectMockProperty = property as KProperty1<Any, Any>
+                injectMockProperties.add(injectMockProperty)
+            }
+        }
+
+        if (!injectMockProperties.isEmpty()) {
```

Comment:
this `nonEmpty` check does not make any sense as first actual `hasNext` of `for` cycle would do totally same check.

Reply:
will remove